gradle/libs.versions.toml

```diff
@@ -3,7 +3,7 @@
 # see https://docs.gradle.org/current/samples/sample_convention_plugins.html#things_to_note
 [versions]
 checker = "3.24.0"
-ideProbe = "0.41.0+30-f11e897b-SNAPSHOT"
+ideProbe = "0.0.0+532-f11e897b-SNAPSHOT"
```

Comment:
Note that your fork doesn't have tags: https://github.com/LukaszKontowski/ide-probe

![image](https://user-images.githubusercontent.com/3383210/186454756-7cb583ae-1015-4398-8f8c-27e2d9c52f78.png)

Probably once the tags are pushed to your fork, this version can be unified (the same version used both for local builds and CI builds)

Reply:
Sure, I will fix it

Reply:
fixed!